lib/elixir_tw/web/static/css/landing.scss

```diff
@@ -1,27 +1,32 @@
-#landing-page {
+.landing-page {
 
-  #page-center:before {
-    content: "";
-    position: fixed;
-    left: 0;
-    right: 0;
-    z-index: -1;
+  .container {
+    height: 100vh;
 
-    display: block;
-    background-image: url("/images/landing-bg.jpg");
-    background-repeat: no-repeat;
-    background-size: cover;
-    width: 100%;
-    height: 100%;
-
-    -webkit-filter: blur(5px);
-    -moz-filter: blur(5px);
-    -o-filter: blur(5px);
-    -ms-filter: blur(5px);
-    filter: blur(5px);
+    .row {
+      height: 100%;
+    }
   }
 
-  #landing-message {
-    margin-top: 40%;
+  h2.constructing {
+    size: 4em;
   }
 }
+
+.landing-page::before {
+  content: '';
+  position: fixed;
+  left: 0;
```

Comment:
Properties should be ordered content, position, right, left, z-index, display, width, height, background-image, filter, background-repeat, background-size

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)